core/src/main/scala/org/apache/spark/scheduler/AsyncEventQueue.scala

```diff
@@ -112,6 +112,7 @@ private class AsyncEventQueue(val name: String, conf: SparkConf, metrics: LiveLi
   private[scheduler] def start(sc: SparkContext): Unit = {
     if (started.compareAndSet(false, true)) {
       this.sc = sc
+      lastReportTimestamp = System.currentTimeMillis()
```

Comment:
According to the description of `lastReportTimestamp`:
```
/** When `droppedEventsCounter` was logged last time in milliseconds. */
```

It seems not very sound to update the val on `start()`.

Reply:
But there's no 'last time' for 'first time'.